### DISCUSSION TITLE
Latest docker version (v1.1.1 and v1.1.0) does not run successfully

### DISCUSSION THREAD

**Discussion by Tavish9** [StanfordVL/BEHAVIOR-1K#986] [Q&A] [answered]
**Describe the bug**
Example Code Fails on v1.1.0 and v1.1.1, but Works on v1.0.0 and Below.

**To Reproduce**
1. Go to [Install with Docker](https://behavior.stanford.edu/omnigibson/getting_started/installation.html#__tabbed_1_2), pull latest docker version and launch.
2. Inside the container, run a simplest example `python -m omnigibson.examples.robots.robot_control_example --quickstart`.
3. See error

**Expected behavior**
No errors thrown in the terminal and the OmniGibson Simulator started successfully.

**Screenshots**
![bug1](https://github.com/user-attachments/assets/d44e7898-2378-4d7b-b8af-e39d6d9c3c3c)
![bug2](https://github.com/user-attachments/assets/4eee7919-3f23-4d70-84cd-56469604cd83)

**Desktop:**
 - OS: Ubuntu 20.04.6
 - Isaac Sim Version: None
 - OmniGibson Version: 1.1.1 and 1.1.0

**Additional context**
In versions v1.0.0 and below, I was able to successfully run the examples provided on the official website. However, after upgrading to v1.1.0 and v1.1.1, I encountered the error shown in the attached screenshot. Other examples also fails.

**Comment by hang-yin** [comment]
@cgokmen Is this related to our recent docker issues?

**Comment by cgokmen** [comment, marked as answer]
Hi, I dont believe this is actually related to docker. It's failing at a point that is too late for anything about Docker to go wrong. Could you please delete and redownload the dataset? I believe the issue is an incomplete dataset unpack.

**Reply by Tavish9** [reply]
Hi, thank you for your prompt response! After re-downloading the dataset, everything is working perfectly now. 

I would like to suggest adding a **checksum** verification for the local dataset to ensure it stays updated when pulling a new version of the Docker image.

**Reply by cgokmen** [reply]
We are working on some kind of versioning scheme for the dataset, definitely. Sorry for the trouble!
